packages/ingest/src/HttpProviderClient.js

```diff
@@ -59,13 +60,11 @@ class HttpProviderClient {
       this.username = this.providerConfig.username;
       this.password = this.providerConfig.password;
     }
-    const cookieJar = new CookieJar();
-    const auth = `${this.username}:${this.password}`;
-    this.gotOptions = {
-      ...this.gotOptions,
-      auth,
-      cookieJar,
-    };
+
+    this.gotOptions.cookieJar = new CookieJar();
+
+    if (this.username) this.gotOptions.username = this.username;
```

Comment:
👍

Reply:
Though I'm not sure if `gotOptions.username` would achieve anything without `password`, so should we test for both before assigning them to gotOptions?

Reply:
Maybe, but that seems like something that should be handled in the `providers` endpoint or model, not here.